### DISCUSSION TITLE
Ability To Use Mobx Stores in getServerSideProps

### DISCUSSION THREAD

**Discussion by DovahBrownies** [vercel/next.js#23309] [Help]
I'd like to be able to user mobx stores inside of my page's `getServerSideProps` function.
I need to do this to access information and functions exclusive to the stores.

Previously, while using `getInitialProps`, I used to do it as such:

__app.js_
```
...
NextJSMobxApp.getInitialProps = async (ctx) => {
  const rootStore = intializeRootStore()
  appContext.ctx.rootStore = rootStore

  let appProps = await App.getInitialProps(appContext) // Pass context down with rootStore

  return {
    ...
  }
}
...
```

_myPage.js_
```
const MyPage = () => {}

MyPage.getInitialProps = async (ctx) => {
  const { rootStore } = ctx

  // Do whatever I want with rootStore here.

  return {
    info: {
      ...
    },
  }
}

export default MyPage
```

**Any idea on how this can be accomplished with `getServerSideProps`?**

**Comment by DovahBrownies** [comment]
Bump. 🥺

Any hints as to how I can achieve anything similar (in `getServerSideProps`) would be appreciated!

**Comment by dmwin72015** [comment]
me too

**Comment by GabenGar** [comment]
Isn't mobx client-only?
